packages/rancher-cis-benchmark/charts/values.yaml

```diff
@@ -8,7 +8,7 @@ image:
     tag: v1.0.8
```

Comment:
@doflamingo721 can we also bump the `cis-operator` image to [v1.0.9](https://github.com/rancher/cis-operator/releases/tag/v1.0.9), please?
I did only an image refresh to retrieve updates and security patches. No additional modification was done.